packages/xrpl/test/models/XChainCreateBridge.test.ts

```
@@ -34,82 +37,45 @@ describe('XChainCreateBridge', function () {
   })
 
   it('verifies valid XChainCreateBridge', function () {
-    assert.doesNotThrow(() => validateXChainCreateBridge(tx))
-    assert.doesNotThrow(() => validate(tx))
+    assertValid(tx)
   })
 
   it('throws w/ missing XChainBridge', function () {
     delete tx.XChainBridge
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)